contracts/Nexus.sol

```diff
@@ -78,8 +78,6 @@ contract Nexus is INexus, BaseAccount, ExecutionHelper, ModuleManager, UUPSUpgra
     }
 
     /// Validates a user operation against a specified validator, extracted from the operation's nonce.
-    /// The entryPoint calls this only if validation succeeds. Fails by returning `VALIDATION_FAILED` for invalid signatures.
```

Comment:
why is this removed?

Reply:
"The entryPoint calls this only if validation succeeds. " This is incorrect as the entryPoint cannot check if validation succeeds before calling "validateUserOp", so remove it.

"Fails by returning `VALIDATION_FAILED` for invalid signatures." This is also incorrect because the function reverts with VALIDATION_FAILED if the validator, which comes from the nonce, is not installed, the signature has nothing to do with the VALIDATION_FAILED revert.

Reply:
So removing is ok

Reply:
@VGabriel45 `validationData = IValidator(validator).validateUserOp(op, userOpHash);`

`VALIDATION_FAILED` is returned also if the module retunes `VALIDATION_FAILED`, which happens when the signature is invalid.

Reply:
#10 
> [Nexus.sol#L81](https://cantina.xyz/code/d1d4b139-9705-4367-9468-297b7078674e/contracts/Nexus.sol#L81): This was confusing to read, I think it was copy pasted from a different part of the code and it doesn't actually apply here. /// The entryPoint calls this only if validation succeeds. Fails by returning VALIDATION_FAILED for invalid signatures.

Reply:
yes, should be corrected